app/helpers/mailer.js

```diff
@@ -0,0 +1,14 @@
+const nodemailer = require('nodemailer');
+const logger = require('../logger');
+const { email: user, pass, service } = require('../../config').common.mailer;
+
+module.exports.sendMail = options => {
+  try {
+    const transporter = nodemailer.createTransport({ service, auth: { user, pass } });
+    transporter.sendMail({ from: user, ...options }, (error, info) => {
```

Comment:
it's preferred to not use callbacks, in this case you have the option to not pass a callback argument and the function will return a a promise object